libsql-replication/src/injector/libsql_injector.rs

```diff
@@ -0,0 +1,45 @@
+use std::mem::size_of;
+
+use libsql_wal::io::StdIO;
+use libsql_wal::replication::injector::Injector;
+use libsql_wal::segment::Frame as WalFrame;
+use zerocopy::{AsBytes, FromZeroes};
+
+use crate::frame::FrameNo;
+use crate::rpc::replication::Frame as RpcFrame;
+
+use super::error::{Error, Result};
+
+pub struct LibsqlInjector {
+    injector: Injector<StdIO>,
+}
+
+impl super::Injector for LibsqlInjector {
+    async fn inject_frame(&mut self, frame: RpcFrame) -> Result<Option<FrameNo>> {
+        // this is a bit annoying be we want to read the frame, and it has to be aligned, so we
+        // must copy it...
+        // FIXME: optimize this.
```

Comment:
are we tracking these anywhere?

Reply:
not really, I'll `grep FIXME` when I find time